installing/Ubuntu.md

```diff
@@ -0,0 +1,44 @@
+# Ubuntu
+
+We chose Ubuntu trusty (14.04) for our example, but newer and older
+versions would work just the same.
+
+First some required dependencies:
+
+```bash
+sudo apt-get install libdbd-sqlite3-perl libdbi-perl libwww-curl-perl
+```
+
+We use the *incredibly dangerous and generally discouraged* way of
```

Comment:
Nix releases binary packages for Ubuntu too: http://hydra.nixos.org/release/nix/nix-1.10 . I think it's ok to leave as is for now though.